src/mutex.rs

```diff
@@ -140,12 +140,26 @@ struct Inner<T: ?Sized> {
     data: UnsafeCell<T>,
 }
 
+/// `MutexWeak` is non-owning reference to a mutex like [`Weak`] for [`Arc`].
```

Comment:
"mutex" in this sentence should be a link.  I suggest the following wording:
```
`MutexWeak` is a non-owning reference to a [`Mutex`].  `MutexWeak` is to `Mutex` as [`std::sync::Weak`] is to [`std::sync::Arc`].
```

Reply:
How I make a link to `Mutex`. The problem is that doc link would depend on the specific like https://docs.rs/futures-locks/0.3.3/futures_locks/struct.Mutex.html.

Reply:
You can make a relative link like this:
```
[`Mutex`]: struct.Mutex.html
```